src/Tribe/Admin/Notice/Timezones.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Shows an admin notice for Timezones
+ * (When using UTC and on TEC Pages or WordPress > General Settings)
+ */
+class Tribe__Events__Admin__Notice__Timezones {
+
+	public function __construct() {
+
+		// Bail if the site isn't using UTC
+		if ( ! $this->is_utc_timezone() ) {
+			return;
+		}
+
+		// Bail if we're not on TEC pages or WordPress General Settings
+		if ( ! $this->should_display() ) {
+			return;
+		}
+
+		tribe_notice(
+			'tribe-events-utc-timezone',
+			array( $this, 'notice' ),
+			array(
+				'type'    => 'warning',
+				'dismiss' => 1,
+				'wrap'    => 'p',
+			)
+		);
+
+	}
+
+	/**
+	 * Checks if the site is using UTC Timezone Options
+	 *
+	 * @since  TBD
+	 *
+	 * @return boolean
+	 */
+	public function is_utc_timezone() {
+
+		// timezone_string
+		$timezone_string = get_option( 'timezone_string' );
+
+		// GMT offset
+		$gmt_offset      = get_option( 'gmt_offset' );
+
+		// If the site is using UTC or UTC manual offset
+		if (
```

Comment:
I think you can simplify the returns here by doing: 

```php
return 'UTC' === $timezone_string || ('' === $timezone_string && '' !== $gmt_offset );
```

So you don't need to specify `true` or `false` for the next returns.

Reply:
Thanks! Mondays... 😂